atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraContainer.java

```diff
@@ -131,16 +133,14 @@ public Refreshable<CassandraKeyValueServiceRuntimeConfig> getRuntimeConfig() {
     }
 
     public CassandraKeyValueServiceConfig getConfigWithProxy(SocketAddress proxyAddress) {
-        Preconditions.checkState(config.servers() instanceof CqlCapableConfig, "Has to be CqlCapableConfig");
-        CqlCapableConfig cqlCapableConfig = (CqlCapableConfig) config.servers();
+        Preconditions.checkState(
+                getRuntimeConfig().get().servers() instanceof CqlCapableConfig, "Has to be CqlCapableConfig");
 
         return ImmutableCassandraKeyValueServiceConfig.builder()
                 .from(config)
-                .servers(ImmutableCqlCapableConfig.builder()
-                        .from(cqlCapableConfig)
-                        .build())
-                .asyncKeyValueServiceFactory(new DefaultCassandraAsyncKeyValueServiceFactory(
-                        new DefaultCqlClientFactory(getClusterBuilderWithProxy(proxyAddress))))
+                .asyncKeyValueServiceFactory(
+                        new DefaultCassandraAsyncKeyValueServiceFactory(new DefaultCqlClientFactory(
+                                () -> Cluster.builder().withNettyOptions(new SocksProxyNettyOptions(proxyAddress)))))
```

Comment:
why did we stop using `getClusterBuilderWithProxy`?

Reply:
Think that got lost when I was resolving merges... will fix!